packages/uploadthing/src/client.ts

```diff
@@ -169,9 +169,20 @@ export const DANGEROUS__uploadFiles = async <TRouter extends FileRouter>(
     // Poll for file data, this way we know that the client-side onUploadComplete callback will be called after the server-side version
     await pollForFileData(presigned.key);
 
+    // TODO: remove `file` prefix in next major version
     return {
-      fileKey: presigned.key,
-      fileUrl: genUrl,
+    // @deprecated - use `name` instead
```

Comment:
Indentation broke. Suggesting on mobile is hard 😅

Reply:
Haha I'll fix it in a minute.